src/ontology/mondo-ingest.Makefile

```diff
@@ -169,6 +169,28 @@ update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+# TODO: Add .PHONY here and below to top of file?
+# TODO: need a purl for this?
+# TODO: use this target: tmp/mondo.sssom.tsv
+.PHONY: mondo-mappings
+MONDO_MAPPINGS_TSV = 'https://raw.githubusercontent.com/monarch-initiative/mondo/master/src/ontology/mappings/mondo.sssom.tsv'
+mondo-mappings:
+	wget ${MONDO_MAPPINGS_TSV} -P ../analysis/excluded_terms_in_mondo/ignored/mappings/
+
+# TODO: Need to add dependency for sig/mirror files?
+.PHONY: excluded_terms_in_mondo
+# TODO: can put two file names as the target names
+# excluded-terms-in-mondo: mondo-mappings
+#     'outpath_xrefed_terms': os.path.join(PROJECT_DIR, 'excluded_terms_in_mondo - xrefs.csv'),
+#     'outpath_in_terms': os.path.join(THIS_SCRIPT_DIR, 'ignored', 'output', 'excluded_terms_in_mondo - in_mondo.csv'),
+# TODO: update my script to accept this positional param $@
+# TODO: change from CSV to TSV output
+# note: why not use $@: will only pick the calling target (e.g. if someone calls for the firs tone)
+$(REPORTDIR)/excluded_terms_in_mondo_xrefs.csv $(REPORTDIR)/excluded_terms_in_mondo.csv: mondo-mappings
+	python3 -m ../scripts/excluded_terms_in_mondo \
```

Comment:
Python scripts embedded like this in a makefile should have all their parameters explicit.. in particular, you should pass in all input files and the paths of the output files. That way we can much more easily understand what is up.

Reply:
Yep, I remember this conversation! I agree. will do

Reply:
I did this for the exclusion tables PR. However, we haven't designed this yet to run on a single ontology; it's meant to run on all ontologies AFAIK.

This is a duplicate of another review thread. I'll resolve this one, but let's continue the discussion there: https://github.com/monarch-initiative/mondo-ingest/pull/35#discussion_r930526547